src/collections/_documentation/apm/distributed-tracing.md

```diff
@@ -0,0 +1,344 @@
+
+If you enable APM collection for a large portion of your traffic, you may exceed your organization's [Quotas and Rate Limits]({%- link _documentation/accounts/quotas.md -%}).
+
+## Setting Up Tracing
+
+{% capture __alert_content -%}
+Sentry's Application Performance Monitoring (APM) features are currently in beta. For more details about APM access, feel free to reach out at [https://sentry.io/for/apm/](https://sentry.io/for/apm/).
+
+Supported SDKs for Tracing:
+- JavaScript Browser ≥ 5.13.0
+- Node.js
+- Python version ≥ 0.11.2
+
+{%- endcapture -%}
+{%- include components/alert.html
+    title="Note"
+    content=__alert_content
+    level="warning"
+%}
+
+### Python
+
+To send any traces, set the `traces_sample_rate` to a nonzero value. The following configuration will capture 10% of all your transactions:
+
+```python
+import sentry_sdk
+    
+sentry_sdk.init("___PUBLIC_DSN___", traces_sample_rate=0.1)
+```
+
+**Automating Traces**
+
+Many integrations for popular frameworks automatically capture traces. If you already have any of the following frameworks set up for Sentry error reporting, you will start to see traces immediately:
+
+- All WSGI-based frameworks (Django, Flask, Pyramid, Falcon, Bottle)
+- Celery
+- Redis Queue (RQ)
+
+[Spans]({%- link _documentation/apm/apm-glossary.md -%}#span) are available for the following operations within a [transaction]({%- link _documentation/apm/apm-glossary.md -%}#transaction):
+
+- Database that uses SQLAlchemy or the Django ORM
+- HTTP requests made with `requests` or the `stdlib`
+- Spawned subprocesses
+
+If you want to enable all relevant transactions automatically, you can use this alternative (currently in alpha):
+
+```python
+import sentry_sdk
+
+sentry_sdk.init("___PUBLIC_DSN___", _experiments={"auto_enabling_integrations": True})
+```
+
+**Managing Transactions**
+
+Let’s say you want to create a [transaction]({%- link _documentation/apm/apm-glossary.md -%}#transaction) for an expensive operation (for example, `process_item`) and send the result to Sentry:
+
+```python
+import sentry_sdk
+
+while True:
+  item = get_from_queue()
+
+  with sentry_sdk.start_span(op="task", transaction=item.get_transaction()):
+      # process_item may create more spans internally (see next examples)
+      process_item(item)
+```
+
+**Adding More Spans to the Transaction**
+
+The next example is called in the `process_item` function from the code snippet above. Our SDK can determine if there is a current open `transaction` and add all newly created [spans]({%- link _documentation/apm/apm-glossary.md -%}#span) as child operations to the `transaction`. Keep in mind; each individual span also needs to be finished; otherwise, it will not show up in the `transaction`.
```

Comment:
```suggestion
The next example contains the implementation of the hypothetical `process_item` function called from the code snippet in the previous section. Our SDK can determine if there is a current open `transaction` and add all newly created [spans]({%- link _documentation/apm/apm-glossary.md -%}#span) as child operations to the `transaction`. Keep in mind that each individual span also needs to be manually finished; otherwise, spans will not show up in the `transaction`.
```

Reply:
@untitaker @HazAT @rhcarvalho Spans don't need to be explicitly finished in the Python SDK as right? Since they're managed using the `with` keyword in Python.

These docs suggests otherwise.

Reply:
Indeed if using the `with` keyword they're close when exiting the block.

We just need to stay clear that `with` blocks is not the only way.

Reply:
I made the change that Alberto suggested above. Is there more content I should be adding in regards to `with` blocks? @rhcarvalho